Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -79,7 +80,7 @@
                                                     infoClass="text_blue" showDetail="true" showSummary="true"
                                                     tooltip="true"/>
 
-                                        <!-- Box für die Bearbeitung der Details -->
+                                        <!-- Box f�r die Bearbeitung der Details -->
```

Comment:
But we don't talk here about comments inside the code files. We say about editing of project data inside the application.